dahu/core/app/scripts/templates/views/workspace/actions/move.html

```diff
@@ -1,7 +1,6 @@
-<div class="move">
```

Comment:
I wanted to make the partial part of the parent form div.
Should I then open a new form div in that template ?